hasher-matcher-actioner/hmalib/lambdas/fetcher.py

```diff
@@ -95,27 +93,12 @@ class ThreatExchangeConfig(HMAConfig):
 def lambda_handler(event, context):
     lambda_init_once()
     config = FetcherConfig.get()
-
-    paginator = dynamodb.meta.client.get_paginator("scan")
-
-    response_iterator = paginator.paginate(
-        TableName=config.collab_config_table,
-        ProjectionExpression=",".join(
-            ("#Name", "privacy_group", "tags", "fetcher_active")
-        ),
-        ExpressionAttributeNames={"#Name": "Name"},
-    )
-
-    collabs = []
-    for page in response_iterator:
-        for item in page["Items"]:
-            if item["fetcher_active"]:
-                collabs.append((item["Name"], item["privacy_group"]))
+    collabs = ThreatExchangeConfig.get_all()
 
     now = datetime.now()
     current_time = now.strftime("%H:%M:%S")
 
-    names = [collab[0] for collab in collabs[:5]]
+    names = [collab.name for collab in collabs[:5]]
```

Comment:
I think we're going to change this so that privacy group name is stored as collab.privacy_group_name. See my comment on  #494

Reply:
I'm just going to follow whatever the final version looks like, you can ignore diff until then.